pennylane_qulacs/qulacs_device.py

```diff
@@ -64,26 +65,6 @@ def _reverse_state(state_vector):
     return reversed_state
 
 
-def _transpose_state_for_wires(state_vector, wires):
```

Comment:
This can be removed since it's now handled by the `_expand_state` function instead.

Reply:
Was this one just added in the previous PR? 😆 

I always find it nice when code can be deleted!

Reply:
Yes, exactly! I should've just fixed both issues at the same time, but I only focused on solving the one issue (didn't know about the other one until later) in the last PR.